### DISCUSSION TITLE
Automatic Failover for Aurora or Patroni

### DISCUSSION THREAD

**Discussion by davecramer** [pgjdbc/r2dbc-postgresql#526] [Ideas]
Are there any plans to implement something like this?

**Comment by mp911de** [comment]
Difficult to answer because I don't know what that would involve. From quickly skimming over the docs, it should be a matter of just using the right DNS name, no?

**Reply by davecramer** [reply]
Sort of, that ends up taking too long. There are other ways. One can query the cluster to see the other instances in the cluster. Also think that is possible in patroni

**Comment by davecramer** [comment]
There is also an API to figure out which server is now the master `aurora_replica_status()`
